code/game/machinery/cryopod.dm

```diff
@@ -241,13 +241,15 @@ GLOBAL_LIST_EMPTY(cryopod_computers)
 		// Eject dead people
 		if(mob_occupant.stat == DEAD)
 			open_machine()
-
-		if(!(world.time > despawn_world_time))
+		if (!mob_occupant.key && mob_occupant.stat < 2) // Occupant is living but logged out entirely.
+			if(!control_computer)
+				find_control_computer(urgent = TRUE)
+			despawn_occupant()
+		if(!(world.time > despawn_world_time)) // AFKs must wait for the proper despawn time.
 			return
-
 		if(!mob_occupant.client && mob_occupant.stat < 2) //Occupant is living and has no client.
```

Comment:
```suggestion
		if((!mob_occupant.client || !mob_occupant.key) && mob_occupant.stat < 2) //Occupant is living and has no client.
```

Now you can remove the lines you added.

Reply:
Well, with this, I want it to despawn instantly if they're SSD (!key) and not dead. AFK's require the normal despawn timer, though